concepticondata/concepticon.tsv

```diff
@@ -3933,4 +3933,27 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3947	HONEYCOMB	Animals	A wax structure consisting of multiple small holes made by bees to store their honey.	Person/Thing	
 3948	SHOTGUN	Warfare and hunting	A long gun (used mainly for hunting) that shoots multiple small bullets at a time.	Person/Thing	
 3949	RED-FOOTED TORTOISE	Animals	A medium-sized species of tortoise (Chelonoidis Carbonarius), found in northern parts of South America.	Person/Thing	
-3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
\ No newline at end of file
+3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
+3951	PANDA	Animals	Also *giant panda*, a white-and-black mammal of the *Ailuropoda* family, native to eastern Asia, commonly referred to as a type of bear.	Person/Thing	
+3952	REFRIGERATOR	Modern world	A storage box, compartment or room to stock goods by means of mechanical refrigeration.	Person/Thing	
+3953	CHARITY	Social and political relations	A benevolent action in favor of the needy, or an activity or gift for the benefit of the public.	Property	
```

Comment:
The ontological category needs to be changed to Person/Thing.

Reply:
The semantic field could also be Possession.

Reply:
Thanks @AnnikaTjuka for your input! So to keep in mind for the future, for instances of nouns referring to general actions (in this case "charity" as a general practice) we consider them as Thing, whereas Property aligns with adjectival/adverbial glosses. In that case, the ontological category should align with the canonical ontology of the gloss' lexical class (nouns ~ entities, verbs ~ processes, etc.), correct? Would this suggest that FRIENDSHIP should also be categorized as Person/Thing?

Reply:
Yes, that's correct. FRIENDSHIP should also be Person/Thing.